packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -354,3 +354,12 @@ export function erc20DataToErc20BridgeToken(data: Erc20Data): ERC20BridgeToken {
     listIds: new Set()
   }
 }
+
+export function getNativeTokenLogo(chainId: number) {
+  switch (chainId) {
+    case ChainId.Xai:
+      return '/images/XaiLogo.svg'
+    default:
+      return 'https://raw.githubusercontent.com/ethereum/ethereum-org-website/957567c341f3ad91305c60f7d0b71dcaebfff839/src/assets/assets/eth-diamond-black-gray.png'
```

Comment:
not introduce by this PR but it seems to make sense to keep the logo in the code base too?